setup.py

```diff
@@ -8,9 +8,9 @@
 import sys
 
 # required to load VERSION_STRING constant
-sys.path.insert(0, './searx')
-from version import VERSION_STRING
-import brand
+sys.path.insert(0, './')
```

Comment:
I think this line can be removed since : 

> As initialized upon program startup, the first item of this list, path[0], is the directory containing the script that was used to invoke the Python interpreter.

( https://docs.python.org/3/library/sys.html#sys.path )

Reply:
Thanks for the tip, I removed the line.